docker/zenml-server-devel.Dockerfile

```diff
@@ -1,30 +0,0 @@
-ARG PYTHON_VERSION=3.9
-FROM python:${PYTHON_VERSION}-slim AS base
-
-ENV PYTHONFAULTHANDLER=1 \
-    PYTHONUNBUFFERED=1 \
-    PYTHONHASHSEED=random \
-    PIP_NO_CACHE_DIR=1 \
-    PIP_DISABLE_PIP_VERSION_CHECK=1 \
-    ZENML_CONTAINER=1
-
-
-WORKDIR /zenml
-
-COPY README.md pyproject.toml ./
-# The existence of this __init__.py file allows the pip install before actually 
-# copying our source files which would invalidate caching
-COPY src/zenml/__init__.py ./src/zenml/
-
-RUN pip install -e .[server]
-COPY src src
-
-RUN mkdir -p .zenconfig/local_stores/default_zen_store
-COPY zenml.db .zenconfig/local_stores/default_zen_store
```

Comment:
Sorry, why this file is deleted?

Reply:
it's not needed anymore. It has been replaced by `zenml-server-dev.Dockerfile`